luke_bot/discord_bot.py

```diff
@@ -1,28 +1,33 @@
 import logging
 
-from discord import app_commands, Intents, Embed, Interaction
-from discord.ext import tasks, commands
+from discord import Embed, Intents, Interaction, app_commands
+from discord.abc import Messageable
+from discord.ext import commands, tasks
 from discord.message import Message
 
-from .settings import settings
-from .smashgg_query import check_luke, get_last_bracket_run, get_last_set
+from .settings import bot_settings
+from .startgg_query import check_luke, get_last_bracket_run, get_last_set
 
 logger = logging.getLogger(__name__)
 
-PLAYER_NAME: str = settings.PLAYER_NAME
-HELP_TEXT = f'I report updates on {PLAYER_NAME} by polling the start.gg API.'
+PLAYER_NAME: str = bot_settings.PLAYER_NAME
+HELP_TEXT = f"I report updates on {PLAYER_NAME} by polling the start.gg API."
 
 
 def same_update(update1: str, update2: str) -> bool:
-    """Checks if 2 blocks of update text contain the same information"""
-    lines1 = update1.strip().splitlines()
-    lines2 = update2.strip().splitlines()
+    """Checks if 2 blocks of update text contain the same information."""
+
+    def clean(update: str) -> list[str]:
+        return update.strip().splitlines()
+
+    lines1 = clean(update1)
+    lines2 = clean(update2)
```

Comment:
mainly a reminder for myself but should probably rename this to something more informative than just lines1 and lines2 😅

Reply:
I've changed this now anyway. I've only renamed it from lines1/2 to old_lines and new_lines so if you want to re-rename things in the future with your own ideas go ahead.